ui/raidboss/raidboss_config.js

```diff
@@ -554,7 +554,7 @@ class RaidbossConfigurator {
           div.appendChild(input);
           input.type = 'text';
           input.step = 'any';
-          input.placeholder = this.base.translate(kMiscTranslations.valueDefault);
+          input.placeholder = `${trig.durationSeconds || 3}${this.base.translate(kMiscTranslations.valueDefault)}`;
```

Comment:
```suggestion
          input.placeholder = `${trig.durationSeconds || 3} ${this.base.translate(kMiscTranslations.valueDefault)}`;
```

Add a space would be more clear.